File: c/3-loops/practice/lp-5.c
#include <stdio.h>

int main() {
    int n, m, num;
    scanf("%d %d %d", &n, &m, &num);

    if (num != 0 && num != 1) {
        printf("Invalid input\n");
        return 0;
    }
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < m; j++) {
            printf("%d ", num);
        }
        printf("\n");
    }

    return 0;
}

File: python/7-lists/practice/lp-5.py
a = list(map(int, input().split()))
b = list(map(int, input().split()))

if any(i < 0 for i in a) or any(j < 0 for j in b):
    print("Invalid input")
else:
    merged = a + b
    print(' '.join(map(str, merged)))